docs/source-zh/rst_source/examples/online_coding.rst

```diff
@@ -0,0 +1,181 @@
+在线代码补全强化学习
+=====================
+
+在线代码补全强化学习（Online Coding RL）是 RLinf 框架中的一个重要应用场景。
+通过与 Continue 等代码编辑器的集成，获取用户对代码补全的偏好反馈，可以实现近乎实时的代码生成和反馈学习，快速提高代码补全的质量，和对齐用户的偏好。
+本示例展示了如何使用 RLinf 框架训练一个能够进行在线代码补全任务的模型。
+
+概述
+----
+
+在线代码补全强化学习系统通过以下方式工作：
+
+1. **实时交互**：系统接收来自 Continue 等编辑器的代码补全请求
+2. **模型推理**：使用训练好的模型生成代码补全建议
+3. **用户反馈**：收集用户对生成代码的接受/拒绝反馈
+4. **在线学习**：基于用户反馈实时更新模型参数
+
+这种实时学习机制使得模型能够快速适应用户的编程习惯和偏好。
+
+运行脚本
+------------------
+
+**环境准备**
+
+
+首先确保您已经安装了 RLinf 框架及其依赖：
+
+.. code-block:: bash
+
+   # 安装额外依赖
+   pip install httpx asyncio
+
+**配置 Continue 集成**
+
+1. **安装 Continue 扩展**
+   
+   由于当前 Continue 未支持上传用户对代码补全的偏好反馈，因此我们修改了 Continue 的源码，支持上传用户对代码补全的偏好反馈。
+   用户可从 `这里 <https://github.com/RLinf/continue>`_ 获取编译好的修改后的 Continue 插件，或自行构建。
+
+   下载编译好的 Continue 插件后，在 VS Code 中安装。
```

Comment:
where can I download the compiled continue? or can you provide the instruction to compile continue from source code?